test_requirements.txt

```diff
@@ -10,7 +10,7 @@ cffi; python_version < '3.10'
 # For testing types. Notes on the restrictions:
 # - Mypy relies on C API features not present in PyPy
 # NOTE: Keep mypy in sync with environment.yml
-mypy==0.981; platform_python_implementation != "PyPy"
+mypy==1.3.0; platform_python_implementation != "PyPy"
 typing_extensions>=4.2.0
```

Comment:
```suggestion
typing_extensions>=4.2.0; python_version < '3.10'
```
maybe?

Reply:
this comment still looks valid, because `typing_extensions` is used for Python 3.9 still. and it's also in `test_requirements.txt`